modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/js/reducers/segmentsExperiences.es.js

```diff
@@ -281,6 +281,12 @@ function createSegmentsExperienceReducer(state, action) {
 					fragmentEntryLinks
 				);
 
+				nextState = setIn(
+					nextState,
+					['hasLockedSegmentsExperiment'],
+					true
```

Comment:
@darquesdev I'm doing this because I'm pretty sure that at the moment the response does not include `hasLockedSegmentsExperiment` in the just created Experience. This is not a big issue, but evidences our _dancing_ types 💃

Reply:
true, so far, when creating a segments experience these are the properties included: "active",  "name", "priority", "segmentsEntryId" and "segmentsExperienceId";